packages/alfa-rules/src/sia-r54/rule.ts

```diff
@@ -0,0 +1,54 @@
+import { Rule, Diagnostic } from "@siteimprove/alfa-act";
+import { Element } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+import { Node } from "@siteimprove/alfa-aria";
```

Comment:
```suggestion
import { Rule, Diagnostic } from "@siteimprove/alfa-act";
import { Node } from "@siteimprove/alfa-aria";
import { Element } from "@siteimprove/alfa-dom";
import { Predicate } from "@siteimprove/alfa-predicate";
import { Err, Ok } from "@siteimprove/alfa-result";
import { Page } from "@siteimprove/alfa-web";
```
Keeping imported packages in alphabetical order.